python/build-wheel-macos-m1.sh

```diff
@@ -84,7 +79,6 @@ for ((i=0; i<${#PY_VERSIONS[@]}; ++i)); do
       TRAVIS_COMMIT=$(git rev-parse HEAD)
     fi
 
-    echo $TRAVIS_COMMIT
```

Comment:
why not keep this?

Reply:
oh i mean i guess we could, it was just an accidental debugging print statement i left in. 

(note we are already printing `echo "TRAVIS_COMMIT variable detected. ray.__commit__ will be set to $TRAVIS_COMMIT"`)